recipes/gnumpy/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "gnumpy" %}
+{% set version = "0.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "070b78ee648c42e3dcd2f56a4308cc26bcaa1de30a4f7c078eb5828050a4ec4f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
```

I would recommend you to replace this part by:
```suggestion
  sha256: 070b78ee648c42e3dcd2f56a4308cc26bcaa1de30a4f7c078eb5828050a4ec4f
```

And you can delete those extra ``jinja`` variables